setup.py

```diff
@@ -24,13 +24,16 @@
 Summary
 -------
 """
+from pathlib import Path
+
 import setuptools
 
-with open("README.md", "r") as fh:
+readme_path = Path("README.md")
+with open(readme_path.absolute().as_posix(), "r") as fh:
     long_description = fh.read()
```

Comment:
This is also from a template, it could be changed, I would suggest that we do it later, everytime we do a change it takes a long time to rerun the tests and the doc changes are urgent.